dali/dfuplus/main.cpp

```diff
@@ -107,6 +107,7 @@ void handleSyntax()
     out.append("        diffkeysrc=<old-key-name>   -- use keydiff/keypatch (src old name)\n");
     out.append("        diffkeydst=<old-key-name>   -- use keydiff/keypatch (dst old name)\n");
     out.append("        multicopy=0|1   -- each destination part gets whole file\n");
+    out.append("        preservecompression=0|1 -- optional, default is 0 (don't preserve compression)\n");
```

Comment:
The biggest problem would occur if they were despraying, and decompressing.

I hadn't spotted that it was only supported for copy.  If so then I think we probably could default it on, but we should check with tony kirk etc. if it is likely to cause a problem.